freqtrade/freqai/data_kitchen.py

```diff
@@ -450,38 +469,42 @@ def principal_component_analysis(self) -> None:
 
         from sklearn.decomposition import PCA  # avoid importing if we dont need it
 
-        n_components = self.data_dictionary["train_features"].shape[1]
-        pca = PCA(n_components=n_components)
+        pca = PCA(0.999)
```

Comment:
Nice this is more computationally efficient compared to stable.

Reply:
And more readable imo :)